v3/fleet/control-proxy.service

```diff
@@ -0,0 +1,31 @@
+[Unit]
+Description=run apigateway as control proxy
+Requires=docker.service
+After=docker.service bootstrap.service zk-health.service
+
+[Service]
+Environment="IMAGE=etcdctl get /images/control-proxy"
+Environment="INTERNAL_CONTROL_ELB=etcdctl get /images/control-proxy"
```

Comment:
It did not work with the environment file set. $($CONTROL_ELB) was not set  - I assume the issue is that the docker run is being shelled out so the env context is different? I dunno. It works as is.

From: Matt Fuller <notifications@github.com<mailto:notifications@github.com>>
Reply-To: adobe-platform/mesos-systemd <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Friday, May 27, 2016 at 1:06 PM
To: adobe-platform/mesos-systemd <mesos-systemd@noreply.github.com<mailto:mesos-systemd@noreply.github.com>>
Cc: Israel Derdik <derdik@adobe.com<mailto:derdik@adobe.com>>, Mention <mention@noreply.github.com<mailto:mention@noreply.github.com>>
Subject: Re: [adobe-platform/mesos-systemd] added control-proxy service (#228)


In v3/fleet/control-proxy.service<https://github.com/adobe-platform/mesos-systemd/pull/228#discussion_r64936807>:

> @@ -0,0 +1,31 @@
> +[Unit]
> +Description=run apigateway as control proxy
> +Requires=docker.service
> +After=docker.service bootstrap.service zk-health.service
> +
> +[Service]
> +Environment="IMAGE=etcdctl get /images/control-proxy"
> +Environment="INTERNAL_CONTROL_ELB=etcdctl get /images/control-proxy"


@iderdik<https://github.com/iderdik> yes, you can reference that, as long as you set your EnvironmentFile above

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/adobe-platform/mesos-systemd/pull/228/files/ed122290c05671a0934d5b9e34e09e89fc78e264#r64936807>, or mute the thread<https://github.com/notifications/unsubscribe/AAyR4ueyxoSFi2i3KTkCDT1kOxeWb3fzks5qFySegaJpZM4IorQr>.